README.md

```diff
@@ -1,8 +1,37 @@
 # How to run
-    1. Install Intelij
+    1. Install IntelliJ
     2. Install the lombok plugin https://projectlombok.org/setup/intellij
     3. Enable 'annotation processing' in settings
     4. Clone project by clicking new->Project from Version Source Control
     5. Import maven project by right clicking on the pom.xml and telling Intelij to import maven project
     6. In the maven tab at the right click the refresh symbol
     7. Click on play
+
+#Implementing a Seeker
```

Comment:
Small one: Markdown uses # Heading instead of #Heading 💯

Reply:
this is already done in the last commit i think